telescope/src/main/java/com/mattprecious/telescope/TelescopeLayout.java

```diff
@@ -59,7 +59,7 @@
  */
 public class TelescopeLayout extends FrameLayout {
   private static final String TAG = "Telescope";
-  static final SimpleDateFormat SCREENSHOT_FILE_FORMAT =
```

Comment:
http://errorprone.info/bugpattern/SimpleDateFormatConstant
It's not a constant, as the name implies.